ding_readers_compass.pages_default.inc

```diff
@@ -1,113 +1,114 @@
-<?php

```

Comment:
Can you just change the lines needed instead of writing the whole file?

Reply:
Don't know why, but this is something I can't control :/

Reply:
It's OK you can't control it - but we should not have diff like this